SPECS/gzip/gzip.spec

```diff
@@ -1,14 +1,18 @@
 Summary:        Programs for compressing and decompressing files
 Name:           gzip
 Version:        1.9
-Release:        4%{?dist}
+Release:        5%{?dist}
 License:        GPLv3+
 URL:            http://www.gnu.org/software/gzip
 Group:          Applications/File
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Source0:	    http://ftp.gnu.org/gnu/gzip/%{name}-%{version}.tar.xz
-%define sha1 gzip=0249ad4c4ca1f144714e8e21b6d0db24651fc122
+
+%if %{with_check}
+BuildRequires:  less
```

Comment:
Since `gzip` is a toolchain package and `less` is not, this will trigger rebuilds on the toolchain during regular package building. On `dev` this is considered a build break.

Either `less` would need to be added to the toolchain build or this check section should be omitted for now.